projects/extension/src/components/IconWeb3.tsx

```diff
@@ -10,10 +11,13 @@ interface Props {
 const hasGlyph = (string: string) =>
   ["kusama", "polkadot", "westend", "rococo"].indexOf(string) > -1
```

Comment:
Will I be allowed to parachains like Astar, Shiden & GM?

My previous PR was reverted.
https://github.com/paritytech/substrate-connect/pull/1315

Reply:
No, only these 4 relay chains have an icon because we have explicitly integrated them into substrate-connect.
We don't want to curate a list of chains like PolkadotJS does.

The reason why these 4 relay chains are hardcoded in the extension is because, specifically for relay chains, it makes syncing them much faster. And because this is a Parity product, we're also skewed towards putting the Parity chains in the code. This optimization, however, is irrelevant for parachains. Hardcoding parachains in the code brings absolutely no technical benefit.
So basically it's these 4 chains and no other.